Core/include/Acts/Vertexing/AdaptiveGridTrackDensity.hpp

```diff
@@ -60,80 +62,66 @@ class AdaptiveGridTrackDensity {
 
   AdaptiveGridTrackDensity(const Config& cfg) : m_cfg(cfg) {}
 
-  /// @brief Returns the z position of maximum track density
+  /// @brief Finds the maximum density of a DensityMap
+  /// @param densityMap Map between z bins and corresponding density value
+  /// @return Iterator of the map entry with the highest density value
+  DensityMap::const_iterator highestDensityEntry(
```

Comment:
Can this be a reference return or do you iterate over the result?

Reply:
I am not sure what you mean, could you please elaborate?

Here is my reasoning:
I use `std::max_element` to find the iterator corresponding to the maximum entry of the map. Since iterators are small objects, I decided to pass it as a copy. 
Is there something I am missing?